ckanext/harvester_dashboard/templates/harvester_dashboard/list.html

```diff
@@ -0,0 +1,37 @@
+{% extends "page.html" %}
+
+{% block subtitle %}{{ _("Harvest sources") }}{% endblock %}
+
+{% block breadcrumb_content %}
+  <li class="active">{{ _("Harvest Dashboard") }}</li>
+{% endblock %}
+
+{% block page_header %}
+{% endblock %}
+
+{% block primary_content_inner %}
+  <section class="module-content">
+    <h1>{{ _('Harvester Dashboard') }}</h1>
+    <p>Last Harvest Jobs ordered by start date</p>
```

Comment:
not sure if ordering by start date is the most helpful for the BFS. Maybe that is worth asking them :)

I could imagine that it is either helpful to always have the same sorting or by `failed` or `errored` if that is performant anyways, though :/

Reply:
Yes I asked them. That is a good idea.